bin/fin

```diff
@@ -2855,6 +2856,68 @@ config_generate ()
 	fi
 }
 
+create_site ()
+{
+	while true; do
+		read -p "Please enter the name of your project (all one word, no spaces or slashes): " projectName
+		#todo, validate this input
+		break
+	done
+	echo ''
+
+	echo -e "What would you like to install?"
+	echo -e "${green}[1]${NC} Drupal 7"
+	echo -e "${green}[2]${NC} Drupal 8"
+	echo -e "${green}[3]${NC} Wordpress"
+	echo -e ""
+	while true; do
+		read -p ": " newSiteVersion
+		case ${newSiteVersion} in
+			1 )
+				newHumanReadableVersion="Drupal 7"
+				newSiteRepoName="drupal7"
+				break
+				;;
+			2 )
+				newHumanReadableVersion="Drupal 8"
+				newSiteRepoName="drupal8"
```

Comment:
Our experience has shown that URLs should be clear and stored in one place along with all the constants or it will be hard to support them.
Could you move site repo names to constants at the top of the fin like this?

```
REPO_DRUPAL7="https://github.com/docksal/drupal7.git"
REPO_DRUPAL8="https://github.com/docksal/drupal8.git"
REPO_WORDPRESS="https://github.com/docksal/wordpress.git"
```

and then use them like this

```
1)
  target_cms="Drupal 8"
  target_repo="$REPO_DRUPAL7"
  break

...
echo -e "Your site will run ${yellow}${target_cms}${NC}"
git clone -b master ${target_repo} ${project_name}

```